team.html

```diff
@@ -368,22 +368,21 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">
                  <div class="card my-2">
                    <img
                       class="card-img-top p-2"
-                      src="img/team/jaredProfile.jpg"
-                      alt="Jared Joo"
+                      src="img/team/matthewProfile.jpg"
+                      alt="Matthew Ozoroski"
                    />
                    <div class="card-body">
-                     <h4 class="card-title officer-name">Jared Joo</h4>
+                     <h4 class="card-title officer-name">Matthew Ozoroski</h4>
                      <h6
                         class="card-subtitle pt-1 pb-3"
                         style="color: #EF4035"
                      >
-                       Communication Chair
+                       Outreach Chair
                      </h6>
                      <p class="card-text">
-                       Jared is a sophomore studying CS at Virginia Tech. He joined WebDVT to
-                       expand his social media presence and get more involved
-                       in web development. In his spare time, he enjoys to swim
-                       and travel.
+                       Matthew is a Junior studying BIT at Virginia Tech. After taking his
```

Comment:
Could you add **"He (or whatever pronoun you prefer) joined WebDVT because..."** right before "After taking..."? This just makes the sentence flow better.